src/sentry/static/sentry/app/components/commitLink.jsx

```diff
@@ -38,16 +44,22 @@ class CommitLink extends React.Component {
         href={commitUrl}
         target="_blank"
       >
-        {repository.provider.id == 'github' && (
-          <InlineSvg src="icon-github" style={{verticalAlign: 'text-top'}} size="14px" />
-        )}
-        {repository.provider.id == 'bitbucket' && (
-          <InlineSvg
-            src="icon-bitbucket"
-            style={{verticalAlign: 'text-top'}}
-            size="14px"
-          />
-        )}
+        {repository.provider.id == 'github' ||
+          (repository.provider.id == 'integrations:github' && (
```

Comment:
Why not do the same thing here? `['github', 'integrations:github'].includes(repository.provider.id) && (`